core/ir/encode_api.h

```diff
@@ -150,21 +150,23 @@ dr_get_isa_mode(void *drcontext);
 
 DR_API
 /**
- * AArch64 Scalable Vector Extension's vector length in bits is one of:
- * 128 256 384 512 640 768 896 1024 1152 1280 1408 1536 1664 1792 1920 2048
+ * - AArch64 Scalable Vector Extension's vector length in bits is one of:
+ *   128 256 384 512 640 768 896 1024 1152 1280 1408 1536 1664 1792 1920 2048
+ * - RISC-V Vector Extension's vector length in bit is from 64 to 65536 in the
+ *   power of 2.
  * Returns whether successful.
  * TODO i#3044: This function will only allow setting vector length if not
- * running on SVE.
+ * running on SVE or RVV.
  */
 bool
-dr_set_sve_vector_length(int vl);
```

Comment:
An API change needs to be documented: please add this as a compatibility-breaking change to the changelist at https://github.com/DynamoRIO/dynamorio/blob/master/api/docs/release.dox#L127

(API additions are also listed in the separate list below that one in that same file -- but since RISCV is not yet officially released no need to add new RISCV-only interfaces there yet.)